src/components/Artist/Artist.css

```diff
@@ -0,0 +1,39 @@
+.Artist {
```

Comment:
Consider arranging your CSS keys in a consistent order--i.e. always put colors or height/width at the top, whatever feels logical to you.